### PR TITLE
Update README.md with new image and messaging

### PR DESCRIPTION
**Pull request by warpaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-(RCL). Because of its distributed nature, the RCL offers transaction immutability
-without a central operator. The RCL contains a built-in currency exchange and its
-path-finding algorithm finds competitive exchange rates across order books
-and currency pairs.
+Ripple connects banks, payment providers and digital asset exchanges via RippleNet to provide one frictionless experience to send money globally.
+
+# What is the XRP Ledger?
+The XRP Ledger is a network of computers, which uses [a consensus algorithm](https://www.youtube.com/watch?v=pj1QVb1vlC0) to atomically settle and record transactions on a secure distributed database. Because of its distributed nature, the XRP Ledger offers transaction immutability without a central operator. The XRP Ledger contains a built-in currency exchange and its path-finding algorithm finds competitive exchange rates across order books and currency pairs.
 
 ### Key Features
```
Comment (mDuo13) [inline README.md]: Seems odd not to mention Interledger compatibility when that's one of our other big advantages. Escrow and PaymentChannels provide key compatibility making it easy for Interledger payments to pass through the XRP Ledger.

File: README.md [hunk 2]
```diff
@@ -19,31 +21,19 @@ and currency pairs.
...
+Ripple connects banks, payment providers, digital asset exchanges and corporates via RippleNet to provide one frictionless experience to send money globally.
 
-As Ripple adoption grows, so do the number of currencies and counterparties.
-Liquidity providers need to maintain accounts with each counterparty for
-each currency – a capital- and time-intensive endeavor that spreads liquidity
-thin. Further, some transactions, such as exotic currency trades, will require
-multiple trading parties, who each layer costs to the transaction. Thin
-liquidity and many intermediary trading parties make competitive pricing
-challenging.
+Ripple is built on the most advanced blockchain technology that is scalable, secure and interoperates different networks, while providing optional access to the world’s fastest and most scalable digital asset for payments, XRP.
```
Comment (mDuo13) [inline README.md]: This use of "Ripple" is confusing. You mean the company?

File: README.md [hunk 3]
```diff
@@ -19,31 +21,19 @@ and currency pairs.
   - Capacity to process the world’s cross-border payments volume
   - Easy access to liquidity through a competitive FX marketplace
 
-## Cross-border payments
-Ripple enables banks to settle cross-border payments in real-time, with
-end-to-end transparency, and at lower costs. Banks can provide liquidity
-for FX themselves or source it from third parties.
+## Global payments
+Ripple connects banks, payment providers, digital asset exchanges and corporates via RippleNet to provide one frictionless experience to send money globally.
```
Comment (mDuo13) [inline README.md]: Given that the XRP Ledger isn't (currently) part of RippleNet, this feels off-topic for a README about `rippled` unless you have a proper lead-in. One could easily misinterpret that to think that the XRP Ledger and RippleNet were one and the same.

File: README.md [hunk 4]
```diff
@@ -19,31 +21,19 @@ and currency pairs.
...
-counterparty risk, or additional operational costs. By using XRP, liquidity
-providers can specialize in certain currency corridors, reduce operational
-costs, and ultimately, offer more competitive FX pricing.
+Built for enterprise use, XRP offers banks and payment providers a highly efficient, scalable, reliable liquidity option to service cross-border payments. Using XRP, banks can source liquidity on demand in real time without having to pre-fund nostro accounts. Ripple can bridge even exotic currency pairs directly through XRP. Payment Providers use XRP to expand reach into new markets, lower foreign exchange costs and provide faster payment settlement. 
+
+xRapid is for payment providers and other financial institutions who want to minimize liquidity costs while improving their customer experience. Because payments into emerging markets often require pre-funded local currency accounts around the world, liquidity costs are high. xRapid dramatically lowers the capital requirements for liquidity. xRapid uses digital assets, such as XRP, to offer on-demand liquidity, which dramatically lowers costs while enabling real-time payments in emerging markets.
```
Comment (mDuo13) [inline README.md]: I'm not sure that the `rippled` README is the best place to introduce xRapid, but if you are going to include it here you need a better lead-in. You can't start talking about what xRapid is _for_ without first saying what xRapid _is_.

Comment (nbougalis) [inline README.md]: Link "xRapid" to https://ripple.com/solutions/source-liquidity/

**Comment by nbougalis** [comment]
Closing this for now.

**Pull request closed without merging**